src/styles/blocks/container.scss

```diff
@@ -0,0 +1,26 @@
+.box {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+
+  &__item {
+    display: grid;
```

Comment:
you may use `flex` here, cause you don't need rows-columns structure to place text in the center 😉